### ISSUE TITLE
Linux logging needs to be flushed

### ISSUE THREAD

**Issue by Fmstrat** [genielabs/HomeGenie#202]
When using the init.d script, HomeGenie is not flushing the log on each entry. This creates logs that stop at random points, such as:

	2015-08-13T16:56:54.5099150-04:00       HomeAutomation.HomeGenie.Automation     1019    Automation Program      Program.Status  Running
	2015-08-13T16:56:54.5123360-04:00       HomeAutomation.HomeGenie.Automation     1019    Automation Program      Program.Status  Idle
	2015-08-13T16:56:54.5884160-04:00       MIG.Gatew

Basically:
- Tailing logs is difficult, because there is a delay in updating
- Logs end at random points until enough log information has buffered to write out

Thanks.

**Comment by genielabs** [comment]
this is a wanted behavior to prevent FS stressing on SD-card media devices

**Comment by Fmstrat** [comment]
How about an option under Maintenance>Logging that allows for instant flushing? I'm running on a flash device, but many of us use a ramdisk for /var/log and /tmp to overcome the above issue and would prefer instant logging if possible. Thanks.

**Comment by genielabs** [comment]
FYI, HG 1.1 is now using NLog which has a bunch of configuration options.
https://github.com/genielabs/HomeGenie/tree/wip-1.1

**Comment by genielabs** [comment]
https://github.com/nlog/NLog/wiki/Configuration-file

**Comment by Fmstrat** [comment]
Nice. I'm guessing by default BufferingWrapper will be turned on?
